core/database_arango.py

```diff
@@ -869,39 +869,6 @@ def swap_link(self):
             time.sleep(ASYNC_JOB_WAIT_TIME)
         self.save()
 
-    # TODO: Consider extracting this to its own class, given it's only meant
-    # to be called by Observables.
-    def get_tags(self) -> List[Tuple["TagRelationship", "Tag"]]:
```

Comment:
Why are you moving this method to YetiModel directly?

Reply:
I'm moving this to YetiTagModel to group methods that make use of the `tag` attribute into a class that declares it. Yeti objects that want to use Tags can inherit from that.